uicomponent-compose/timetable2021/src/main/res/drawable/toolbar_droidkaigi_2021_logo.xml

```diff
@@ -0,0 +1,57 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Probably it is a little big as Vector Drawable, so it may be better to make it an image such as png, but I will make an issue separately. 👍